core/tls.cabal

```diff
@@ -160,6 +160,7 @@ Benchmark bench-tls
                    , QuickCheck >= 2
                    , tasty-quickcheck
                    , tls
+  ghc-options:       -Wall -fno-warn-unused-imports -fno-warn-orphans
```

Comment:
In general, only `-Wall` should be specified for `ghc-options` ideally. '-fno-warn-unused-imports` is acceptable during the transition phase. But other flags should be file specific, in my opinion.